llvm/lib/Target/ARM/ARMLowOverheadLoops.cpp

```diff
@@ -1035,13 +1050,7 @@ bool LowOverheadLoop::ValidateLiveOuts() {
   SmallVector<MachineInstr *> Worklist(LiveOutMIs.begin(), LiveOutMIs.end());
   while (!Worklist.empty()) {
     MachineInstr *MI = Worklist.pop_back_val();
-    if (MI->getOpcode() == ARM::MQPRCopy) {
-      VMOVCopies.insert(MI);
```

Comment:
So, can we now completely remove VMOVCopies?

(And, not to be part of this patch, but I wonder if the MQPRCopy instruction could also be removed.)